scripts/anvil_tools/post_workspace_attributes.py

```diff
@@ -44,34 +43,60 @@ def setup_single_data_delivery_workspace(request, project="anvil-datastorage"):
         return workspace_dict
 
     workspace_dict["publish_workspace_status"] = "Success"
+    workspace_dict["final_workspace_status"] = "Success"
     return workspace_dict
 
 
-def setup_data_delivery_workspaces(tsv):
-    """Create array of json requests per row in tsv file."""
+def format_dataframe(workspace_row):
```

Comment:
this function name is misleading because you call it on a row, not the whole dataframe, right? maybe change the name, or have the function loop through all rows itself?